plugins/main/tap/distro/rhel/rhel.go

```diff
@@ -0,0 +1,87 @@
+package rhel
+
+import (
+	"fmt"
+	"github.com/containernetworking/plugins/plugins/main/tap/distro"
+	"github.com/opencontainers/selinux/go-selinux"
+	"os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+var Rhel distro.Distro = CreateLink{}
+
+type CreateLink struct{}
+
+func (l CreateLink) CreateLink(tmpName string, mtu int, nsFd int, nsPath string, multique bool, mac string, owner int, group int) error {
+
+	err := setContainerSeBool()
+	if err != nil {
+		return err
+	}
+	err = createSelinuxTap(tmpName, mtu, nsFd, nsPath, multique, mac, owner, group)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func setContainerSeBool() error {
+	output, err := exec.Command("cat", "/sys/fs/selinux/booleans/container_use_devices").CombinedOutput()
```

Comment:
@mmirecki does this come from the selinux userland utilities? Is it common across all distros that use selinux, or is it RHEL specific?

Reply:
Also, can't we just read this file with `ioutil.ReadFile()` or something? Why do we need to shell out to `cat`?

Reply:
@dbcw - Sebools are a selinux concept, so would be there on any distro with selinux
As to the mount point for selinuxfs it is /sys/fs/selinux on RHEL, I have no experience with selinux on other distro's (and the docs are not quite verbose on this).

Changed to os.ReadFIle()